src/Representations/ANF.elm

```diff
@@ -147,3 +150,10 @@ splitANFAndHelp formula =
 sortANFList : List (List String) -> List (List String)
 sortANFList list =
     List.sortBy (\a -> String.fromInt (List.length a) ++ List.foldr (++) "" a) list
+
+
+postProcessANF : String -> String
```

Comment:
Also use this in the Adequacy feature when displaying ANFs.

Reply:
Resolved by 7c41533.